drivers/media/i2c/imx258.c

```diff
@@ -700,16 +720,29 @@ static int imx258_write_regs(struct imx258 *imx258,
 	return 0;
 }
 
+/* Get bayer order based on flip setting. */
+static u32 imx258_get_format_code(struct imx258 *imx258)
+{
+	unsigned int i;
+
+	lockdep_assert_held(&imx258->mutex);
+
+	i = (imx258->vflip->val ? 2 : 0) |
```

Comment:
`REG_CONFIG_MIRROR_VFLIP` (and `_HFLIP` below)?

Reply:
Actually no. The mapping is coincidence.

In this case we're wanting the index into ```codes``` array where the index is defined as 
```
/*
 * The supported formats.
 * This table MUST contain 4 entries per format, to cover the various flip
 * combinations in the order
 * - no flip
 * - h flip
 * - v flip
 * - h&v flips
 */
```

```REG_CONFIG_MIRROR_VFLIP``` and _HFLIP are used in imx258_set_ctrl when setting the register in the sensor.